backends-velox/src/test/scala/org/apache/spark/sql/execution/VeloxLocalCacheSuite.scala

```diff
@@ -34,7 +34,7 @@ class VeloxLocalCacheSuite extends VeloxWholeStageTransformerSuite {
       .set(VeloxConfig.COLUMNAR_VELOX_FILE_HANDLE_CACHE_ENABLED.key, "true")
   }
 
-  testWithSpecifiedSparkVersion("read example parquet files", Some("3.5"), Some("3.5")) {
+  testWithSpecifiedSparkVersion("read example parquet files", "3.5", "3.5") {
```

Comment:
this should change to `testWithMinSparkVersion("read example parquet files", "3.5")`?